src/goto-instrument/code_contracts.h

```diff
@@ -104,6 +104,12 @@ class code_contractst
   /// \brief Enforce contract of a single function
   bool enforce_contract(const std::string &);
 
+  /// \brief Create code corresponding to assertion/assumption
```

Comment:
This comment could be reworded for clarity.
1. Is it supposed to work only for assertions / assumptions?
2. It creates goto instructions (not `codet`), so saying "create code" might be confusing.

Meta comment about the approach: the function seems to be creating a `goto_convert` and just making a single call to it. Can we make this `converter` a private member of the `code_contractst` class instead, and just call the `converter.goto_convert` directly?

Reply:
I update the comment as suggested, but I don't see how adding `converter` as a member of this class would be beneficial here.

Reply:
I meant that `converter` is being initialized with the same 2 arguments:

```
goto_convertt converter(symbol_table, log.get_message_handler());
```

so why not make it a `private`/`protected` member, initialize it within `code_contractst` constructor, and directly call `converter.goto_convert` in place of `convert_to_goto`?

But we only call it twice anyway, so not a big deal.